File: B/C/b008.c
#include <stdio.h>
int main() {
    int incom[1000];
    int M,N,max=0,sum;
    int i,j,n;

    scanf("%d %d", &N, &M);
    if (M>0 && N>0) {
        for (i=0; i<M; i++) {
            sum = 0;
            for (j=0; j<N; j++) {
                scanf("%d", &n);
                sum += n;
            }
            incom[i] = sum;
        }
        max = 0;
        for (i=0; i<M; i++) {
            if (incom[i]>0) {
                max += incom[i];
            }
        }
    }
    printf("%d\n",max);
    return 0;
}

File: B/C#/b008.cs
using System;
using System.Collections.Generic;
namespace Solution {
    class Solution {
        static void Main(string[] args) {

            int max = 0;
            var incom = new List<int>();

            string[] s = System.Console.ReadLine().Trim().Split(' ');
            int N = Int32.Parse(s[0]);
            int M = Int32.Parse(s[1]);
            if (M>0 && N>0)
            {
                for (int i=0; i<M; i++)
                {
                    int sum = 0;
                    string[] line = System.Console.ReadLine().Trim().Split(' ');
                    foreach(var n in line) { sum += Int32.Parse(n); }
                    incom.Add(sum);
                }
                foreach(var n in incom)
                {
                    if (n>0) { max += n; }
                }
            }
            System.Console.WriteLine(max);
        }
    }
}
